scholarmarkdown.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
   s.required_rubygems_version = Gem::Requirement.new(">= 0".freeze) if s.respond_to? :required_rubygems_version=
   s.require_paths = ["lib".freeze]
   s.authors = ["Ruben Taelman".freeze]
-  s.date = "2022-08-29"
```

Comment:
The changes to this file are not necessary. I regen them upon release.

Reply:
That's automatic on `rake gemspec`, should I discard all changes in this file?

Reply:
Yes please :-)